assets/scripts/script.js

```diff
@@ -273,6 +273,7 @@
           .children('.woocommerce-product-gallery__image:not(.has-video)')
           .first();
         $firstGalleryElement.detach().insertAfter(firstImage);
+        setVideoPlayerUrl(1);
```

Comment:
why this is not zero based? in order to be the first one...

Reply:
@chrisbaltazar — it's complicated, if you check the comment above, this will first prepend the product featured image to the gallery images, but then if we have a video it will trigger the slider to swipe automatically it (so position 0 is the featured image, and position 1 is the video) (no clue why, since the docs do not mention any reason). I have tested on my local and it is fixing the issue.

Reply:
Very well then